<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>charts</class>
 <widget class="QWidget" name="charts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>393</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff> icon.ico</normaloff> icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QCustomPlot" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">borde-bottom-right-radius: 10px</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="charts_param_box">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(126, 125, 128, 26);
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-left-radius: 10px;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="min_x">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton{ 
	background-color: None;
}

QRadioButton::indicator::unchecked{ 
	border: 1px solid; 
	border-color: rgba(255, 255, 255, 0);
	border-radius: 7px;
	background-color: white; 
	width: 12px; 
	height: 12px; 
}

QRadioButton::indicator::checked{ 
	border: 1px dotted; 
	border-color: rgba(162, 162, 162);
	border-radius: 7px;
	background-color: rgb(64, 134, 247);
	width: 12px; 
	height: 12px; 
}

</string>
        </property>
        <property name="text">
         <string>min :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="show_min">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>15</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
borde-bottom-right-radius: 10px;
borde-bottom-left-radius: 10px;
color: black;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="max_x">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton{ 
	background-color: None;
}

QRadioButton::indicator::unchecked{ 
	border: 1px solid; 
	border-color: rgba(255, 255, 255, 0);
	border-radius: 7px;
	background-color: white; 
	width: 12px; 
	height: 12px; 
}

QRadioButton::indicator::checked{ 
	border: 1px dotted; 
	border-color: rgba(162, 162, 162);
	border-radius: 7px;
	background-color: rgb(64, 134, 247);
	width: 12px; 
	height: 12px; 
}

</string>
        </property>
        <property name="text">
         <string>max :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="show_max">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
borde-bottom-right-radius: 10px;
borde-bottom-left-radius: 10px;
color: black;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QRadioButton" name="interval_x">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton{ 
	background-color: None;
}

QRadioButton::indicator::unchecked{ 
	border: 1px solid; 
	border-color: rgba(255, 255, 255, 0);
	border-radius: 7px;
	background-color: white; 
	width: 12px; 
	height: 12px; 
}

QRadioButton::indicator::checked{ 
	border: 1px dotted; 
	border-color: rgba(162, 162, 162);
	border-radius: 7px;
	background-color: rgb(64, 134, 247);
	width: 12px; 
	height: 12px; 
}

</string>
        </property>
        <property name="text">
         <string>interval :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="show_interval">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
borde-bottom-right-radius: 10px;
borde-bottom-left-radius: 10px;
color: black;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="charts_operand_box">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>96</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>96</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(126, 125, 128, 26);
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-left-radius: 10px;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="seven">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="eight">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="nine">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="four">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="five">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="six">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="one">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="two">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="three">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="dot">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="zero">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="del">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color:rgb(236, 106, 94);
}</string>
        </property>
        <property name="text">
         <string>del</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="chars_operators_box">
     <property name="minimumSize">
      <size>
       <width>96</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>96</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(126, 125, 128, 26);
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-left-radius: 10px;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="build">
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
 	color: rgb(244, 191, 79);
}</string>
        </property>
        <property name="text">
         <string>build</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="minus">
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
 	color: rgb(244, 191, 79);
}</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
